Xamarin.Forms.Controls/XamStore/Views/SearchHandlerPage.cs

```diff
@@ -37,6 +37,8 @@ public SearchHandlerPage()
 					new ScrollView { Content = new StackLayout {
 														Children = {
 																	new Button { Text = "Show/Hide SearchHandler", Command = new Command(()=> ShowHideSearchHandler()) },
+																	new Button { Text = "Focus/Unfocus SearchHandler", Command = new Command(()=> FocusUnfocusSearchHandler()) },
```

Comment:
This button gets focus when click. SearchHandler always loses focus, at the moment of the command execution. Or not?

Reply:
Hi @paymicro, @rmarinho, @PureWeen. According to Issue #7956 the focus / unfocus events on the SearchHandler were not working for Android. I've placed the PR #14375 that may fix that.